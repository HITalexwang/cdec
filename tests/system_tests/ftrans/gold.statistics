-lm_nodes 13
-lm_edges 16
-lm_paths 4
-lm_trans CB
-lm_viterbi 101
