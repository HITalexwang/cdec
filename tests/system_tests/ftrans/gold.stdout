0 ||| CB ||| F4=1 Inv=1 ||| 101
0 ||| BC ||| F3=0.4 Mono=1 ||| 6
0 ||| B C ||| F1=1 F2=0.2 OtherFeat=1 Mono=1 ||| 3
0 ||| C B ||| F1=1 F2=0.2 OtherFeat=1 Inv=1 ||| 2
